src/workspacer/workspacer.csproj

```diff
@@ -18,26 +18,36 @@
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.Gap\$(OutDir)workspacer.Gap.dll" DestinationFolder="$(TargetDir)plugins\workspacer.Gap\" />
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.ActionMenu\$(OutDir)workspacer.ActionMenu.dll" DestinationFolder="$(TargetDir)plugins\workspacer.ActionMenu\" />
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.FocusIndicator\$(OutDir)workspacer.FocusIndicator.dll" DestinationFolder="$(TargetDir)plugins\workspacer.FocusIndicator\" />
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.Watcher\$(OutDir)workspacer.Watcher.dll" DestinationFolder="$(TargetDir)" />
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.Watcher\$(OutDir)workspacer.Watcher.exe" DestinationFolder="$(TargetDir)" />
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.Watcher\$(OutDir)workspacer.Watcher.pdb" DestinationFolder="$(TargetDir)" />
-  </Target>
-  <Target Name="CopyPluginsAfterPublish" AfterTargets="Publish">
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.Bar\$(OutDir)workspacer.Bar.dll" DestinationFolder="$(PublishDir)plugins\workspacer.Bar\" />
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.Gap\$(OutDir)workspacer.Gap.dll" DestinationFolder="$(PublishDir)plugins\workspacer.Gap\" />
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.ActionMenu\$(OutDir)workspacer.ActionMenu.dll" DestinationFolder="$(PublishDir)plugins\workspacer.ActionMenu\" />
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.FocusIndicator\$(OutDir)workspacer.FocusIndicator.dll" DestinationFolder="$(PublishDir)plugins\workspacer.FocusIndicator\" />
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.Watcher\$(OutDir)workspacer.Watcher.dll" DestinationFolder="$(PublishDir)" />
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.Watcher\$(OutDir)workspacer.Watcher.exe" DestinationFolder="$(PublishDir)" />
-    <Copy SourceFiles="$(SolutionDir)\src\workspacer.Watcher\$(OutDir)workspacer.Watcher.pdb" DestinationFolder="$(PublishDir)" />
-  </Target>
   <ItemGroup>
     <ProjectReference Include="..\workspacer.Native\workspacer.Native.csproj" />
     <ProjectReference Include="..\workspacer.Shared\workspacer.Shared.csproj" />
+
+    <!-- Build-time dependencies -->
+    <ProjectReference Include="..\workspacer.Bar\workspacer.Bar.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\workspacer.Gap\workspacer.Gap.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\workspacer.ActionMenu\workspacer.ActionMenu.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\workspacer.FocusIndicator\workspacer.FocusIndicator.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\workspacer.Titlebar\workspacer.Titlebar.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\workspacer.Watcher\workspacer.Watcher.csproj" ReferenceOutputAssembly="false" />
   </ItemGroup>
```

Comment:
This section looks new, and mostly unrelated to the change at hand. What specifically does it change/improve?

Reply:
It is this commit https://github.com/workspacer/workspacer/pull/246/commits/f290558aa3e25fa9116a65be417b1a9b64699928. It adds build-time dependencies so they are ensured to be built first so the CI succeeds.

Reply:
Ok. Fair enough. Let's keep it :)